nss-ice-phantoms/BangazonAPI
# Computer, Department and Training Program controllers and tests complete

# Description

Completed controllers and tests for Computer, Department and Training Program.

Fixes # (issue)

N/A

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout rr-firstBranch`
3. Verify functionality according to ticket numbers 7 , 8 & 9.
4.  Run tests in the following order Get, Create, Update, Delete.


# Checklist:
